### PR TITLE
MGMT-10210 : Deprecate Bare Metal Assets/ BM IPI in ACM

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://issues.redhat.com/browse/MGMT-10210

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -160,6 +160,8 @@
     "bareMetalAsset.tableHeader.namespace": "Namespace",
     "bareMetalAsset.tableHeader.role": "Role",
     "bareMetalAsset.tableHeader.status": "Status",
+    "bareMetalAsset.warning.text": "The Bare Metal flow will is deprecated and will be removed in the next feature. For Bare Metal cluster consider the assisted installation flows.",
```
Comment (matthewcarleton) [inline frontend/public/locales/en/translation.json:163]:
```suggestion
    "bareMetalAsset.warning.text": "The Bare Metal Assets flow is deprecated and will be removed in the next feature. For Bare Metal clusters consider the assisted installation flows.",
```

**Review by jtomasek — APPROVED** [review]
/lgtm

**Pull request merged**